lms/templates/ccx/student_admin.html

```diff
@@ -2,10 +2,19 @@
 
 <section>
   <h2>${_('Student Grades')}</h2>
-  <p>
-    <a href="${gradebook_url}">${_('View gradebook')}</a>
-  </p>
-  <p>
+  <p class="ccx-grade-book-download">
+    <i class="fa fa-download" aria-hidden="true"></i>
     <a href="${grades_csv_url}">${_('Download student grades')}</a>
   </p>
+  <br/>
+  <div>
+    <iframe src='${gradebook_url}' id="ccx_grade_book_iframe"
+            name="ccx_grade_book_iframe"
+            frameBorder="0"
+            scrolling="no"
+            height="360px"
+            class="ccx-grade-book">
+      <p>Iframe not supported</p>
+    </iframe>
```

Comment:
I don't understand. This seems to defeat the whole purpose of pagination,
which is to only load a small number of students at once, because loading
students triggers a full regrade of each student and is very expensive.

On Wed, Nov 18, 2015 at 7:19 AM, Amir Qayyum Khan <notifications@github.com>
wrote:

> In lms/templates/ccx/student_admin.html
> <https://github.com/edx/edx-platform/pull/10637#discussion_r45192127>:
>
> >      <a href="${grades_csv_url}">${_('Download student grades')}</a>
> >    </p>
> > +  <br/>
> > +  <div>
> > +    <iframe src='${gradebook_url}' id="ccx_grade_book_iframe"
> > +            name="ccx_grade_book_iframe"
> > +            frameBorder="0"
> > +            scrolling="no"
> > +            height="360px"
> > +            class="ccx-grade-book">
> > +      <p>Iframe not supported</p>
> > +    </iframe>
>
> @sarina <https://github.com/sarina> any suggestion?
> The goal of this PR was to minimize number of clicks to get grade book of
> ccx.
> Now grade book has limit 20 students per page that why i embedded grade
> book here in iframe so that ccx coach can see his grade book in 2 one click
> from his dashboard.
>
> Second pagination handling is easy in iframe, when user click next or
> previous page only iframe is refresh instead of whole student admin page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10637/files#r45192127>.
>